.vscode/settings.json

```diff
@@ -17,7 +17,9 @@
         "xstddef": "c",
         "type_traits": "c",
         "utility": "c",
-        "ranges": "c"
+        "ranges": "c",
```

Comment:
This should be reverted.  (and added to a different PR, if it's actually useful/needed)/